### ISSUE TITLE
**Bug** Profile Picture does not update when image changes (but URL stays the same)

### ISSUE THREAD

**Issue by adamsrosco** [AppLozic/Applozic-iOS-SDK#397]
## Goals
See contacts updated profile pictures

## Expected Results
When a contact changes updates their profile picture (custom image url being set), i expect to see the updated image.

## Actual Results
The old cached image is always used, even when updating the image mulitple times on the server.

## Steps to Reproduce
1. Store an image on a server ( for example 'http://www.somesite.com/profileImageA.png' ).
2. Create and insert an ALContact Record with the contactImageURL set to Stored Image 'http://www.somesite.com/profileImageA.png'.
3. Open app and navigate to the chat/contacts section.
4. Observer application cache's and displays Stored Image.
5. Replace Image on server, but keep the same name (i.e. the url does not change).
6. In the app, refresh the view (either go out and back in, or check the contacts screen and scroll).
7. Observe: Image does not update to the latest image that is on the server. 

This issue is actually part of the SDImageCache framework. There is an option for checking http headers of the URL's to indicate whether or not a particular resource has changed, in order to force a download of the new assets.

The imageView's _**sd_setImageWithURL**_ method should include the option _**SDWebImageRefreshCached**_ . This will check the http headers associated with the asset, and update if the asset has changed.

###### Current Example
```objective-c
// ALMessagesViewController.m

-(void)updateProfileImageAndUnreadCount:(ALContactCell *)contactCell WithChannel:(ALChannel*) alChannel orChannelId:(ALContact*)contact{
...
    if(alChannel)
    {
    ...
        if(alChannel.type == GROUP_OF_TWO)
        {
        ...
            if(grpContact.contactImageUrl.length)
            {
                NSURL * theUrl1 = [NSURL URLWithString:grpContact.contactImageUrl];
                [contactCell.mUserImageView sd_setImageWithURL:theUrl1];
            }
        }
    }
}
```

###### Should Be

```objective-c
    NSURL * theUrl1 = [NSURL URLWithString:grpContact.contactImageUrl
    [contactCell.mUserImageView sd_setImageWithURL:theUrl1 placeholderImage:nil options:SDWebImageRefreshCached];
```

**Comment by adamsrosco** [comment]
Hi,

Could this be a feature for a future release perhaps? 

Regards,
Rosco.

**Comment by mukeshthawani** [comment]
Merged #398
